proto/sentry_protos/snuba/v1/endpoint_trace_item_stats.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceItemStatsRequest {
+  RequestMeta meta = 1;
+
+  // filter out trace items you dont want
+  TraceItemFilter filter = 2;
+
+  uint32 limit = 3;
+  uint32 limit_keys_by = 4;
+
+  // optional, used for pagination, the next page token will be returned in the response
+  PageToken page_token = 5;
+}
+
+message TraceItemStats {
+  enum AttributeType {
+    ATTRIBUTE_TYPE_UNSPECIFIED = 0;
+    ATTRIBUTE_TYPE_STRING = 1;
+    ATTRIBUTE_TYPE_NUMERIC = 2;
+  }
+
+  message AttributeResults {
+    string label = 1;
+    float value = 2;
+  }
+
+  string aggregation = 1;
+  string attribute_key = 2;
+  AttributeType attribute_type = 3;
+  repeated AttributeResults data = 4;
+}
+
+// this is a response from the TraceItemStats endpoint
+// it is the counterpart to TraceItemStatsRequest
+message TraceItemStatsResponse {
```

Comment:
https://www.notion.so/sentry/TraceItemStatsResults-Examples-1848b10e4b5d801fa6eff12715876728
here are some examples for how this could evolve long term